README.md

```diff
@@ -47,3 +67,78 @@ To run this on AWS Lambda, see
 This repo uses semver, and its public interface (in order to determine what
 changes would break backwards-compatibility) is defined as the functions
 exported by `./index.js`. 
+
+## Checking PHP versions
+
+Each of the commands for scanning one or more repos also accepts an optional URL
+to a CSV file with mapping information between Docker images and PHP versions.
+
+### Docker-PHP CSV file
+
+Example CSV content:
+
+```
+Docker image,PHP version
+openjdk:8-jdk-alpine,NONE
+php:7.3-apache-buster,7.3
+```
+
+**Note**:
+
+- The header line needs to use exactly those values.
+- The values in the Docker image column should be the exact value used as the
+  `FROM` in the Dockerfile.
+- For Docker images that do include PHP, specify only the major and minor
+  version (such as `7.3`, **not** `7.3.24`).
+- For Docker images that do not include PHP, use `NONE`.
+
+**Tip**:  
+One easy way to maintain a URL-accessible CSV file is as a Google Sheet, using
+the "File" > "Publish to Web" feature, selecting the desired sheet (tab),
+specifying "CSV" as the format option, and using the given URL in calls to this
+library.
+
+### Missing Docker image values
+
+If your list of vulnerabilities includes a warning like the following...
+```
+No record found in spreadsheet for php:7.3-apache-buster
+```
+... you simply need to add a row to your CSV file with that Docker image (in
+this case, `php:7.3-apache-buster`) and what version of PHP is uses (in this
+case, `7.3`).
+
+If you do not know what version of PHP it uses (and if it is a Docker image you
+trust enough to run on your local computer), you can run a command like this,
+replacing `YOUR-DOCKER-IMAGE-STRING` with the actual value:
+```
+docker run -it YOUR-DOCKER-IMAGE-STRING php -v
+```
+
+In the example above, that would mean running the following command:
+```
+docker run -it php:7.3-apache-buster php -v
+```
+
+Note: If the Docker image has an Entrypoint that interferes with the above
+command, try this one, replacing `YOUR-DOCKER-IMAGE-STRING` with the actual
+value:
+```
+docker run -it --entrypoint php YOUR-DOCKER-IMAGE-STRING -v
+```
```

Comment:
Is it appropriate to mention the get-php-of-docker.sh script that does this check?

Reply:
Okay, done.